src/client/theme-default/components/VPNavBarTitle.vue

```diff
@@ -22,14 +22,13 @@ const { hasSidebar } = useSidebar()
 <style scoped>
 .VPNavBarTitle {
   flex-shrink: 0;
-  border-bottom: 1px solid transparent;
```

Comment:
This was adding to the height of the title on small screens, making its height 57px instead of 56px, which made the navbar height 58px instead of 57px, etc.

Since we only mean to have a border on large screens, let's define the border's space only on large screens.